phpunit/functional/TicketTest.php

```diff
@@ -6416,7 +6416,27 @@ protected function timelineItemsProvider(): iterable
 
         $postonly_user_id = getItemByTypeName(\User::class, 'post-only', true);
         $normal_user_id   = getItemByTypeName(\User::class, 'normal', true);
-        $tech_user_id     = getItemByTypeName(\User::class, 'tech', true);
+        $tech_user     = getItemByTypeName(\User::class, 'tech');
+        $tech_user_id = $tech_user->getID();
+
+        $tech_p = getItemByTypeName(\Profile::class, 'Technician');
+        $new_tech = $tech_p->clone();
+
+        $profileright = new \ProfileRight();
+        $profileright->update([
+            'profiles_id' => $new_tech,
+            'name'        => 'task',
+            'rights'      => 5137
+        ]);
+
+        $new_user_id = $tech_user->clone();
+        $user = new \User();
+        $user->update([
+            'id' => $new_user_id,
+            'login' => 'new_user',
+            'password' => 'new_user',
+            'profiles_id' => $new_tech
+        ]);
```

Comment:
I'm not sure the clone is essential?
```suggestion
        $this->createItem('User', [
            'name' => 'new_user',
            'login' => 'new_user',
            'password' => 'new_user',
            'password2' => 'new_user',
            '_profiles_id' => $new_tech,
            '_entities_id' => getItemByTypeName('Entity', '_test_root_entity', true),
        ]);
```